lib/Vend/Util.pm

```diff
@@ -2233,20 +2233,49 @@ sub send_mail {
 	}
 
 	SMTP: {
-		my $mhost = $::Variable->{MV_SMTPHOST} || $Global::Variable->{MV_SMTPHOST};
-		my $helo =  $Global::Variable->{MV_HELO} || $::Variable->{SERVER_NAME};
+		my $cc = $Vend::Cfg->{SMTPConfig};
+		my $gc = $Global::SMTPConfig;
+
+		my $mhost = $cc->{host} || $gc->{host} || $::Variable->{MV_SMTPHOST} || $Global::Variable->{MV_SMTPHOST};
 		last SMTP unless $none and $mhost;
+
 		eval {
 			require Net::SMTP;
 		};
-		last SMTP if $@;
+		logError("Unable to load Net::SMTP module"), last SMTP if $@;
```

Comment:
Where does logError() go during startup? Does it end up at stdout the way logGlobal() does? This seems like it would be good information to see in the terminal during startup, along frankly with the contents of $@.

Reply:
Yeah, I think at startup all the logs go to stderr.

It would be good to check this at startup, but I'm not willing to rearchitect this whole thing now to do that. I was just adding logging here for the otherwise opaque way Net::SMTP failures were historically logged, but I don't want to change *when* the error is raised as part of this SMTP AUTH feature.

You're totally right that `$@` should be included in the message, so I just pushed a revised commit that does that.